### PR TITLE
Refactor: TSDB CLI Changes (UPDATE #4)

### PR DESCRIPTION
**Pull request by imakhlin**
Default CLI config file & template names
- [x ] v3io.yaml -> v3io-tsdb-config.yaml
- [x ] v3io.yaml.template -> v3io-tsdb-config.yaml.template

Schema file keys names
- [x ] Aggregators -> Aggregates
- [x ] AggregatorsGranularity -> AggregationGranularity
- [x ] shardingBuckets -> shardingBucketsCount

Label names
- [x ] "Aggregator" -> "Aggregate"

Environment-variable names
- [x ] V3IO_CONF -> V3IO_TSDB_CONFIG

### CODE REVIEW

**Comment by imakhlin** [comment]
Looks like query performance in CI is too slow - unable to complete data validation on time

```
goroutine 59 [chan receive]:
github.com/v3io/v3io-tsdb/pkg/utils.(*AsyncItemsCursor).NextItem(0xc420159700, 0x0, 0x0, 0x0)
	/home/travis/gopath/src/github.com/v3io/v3io-tsdb/pkg/utils/asynciter.go:154 +0x178
github.com/v3io/v3io-tsdb/pkg/utils.(*AsyncItemsCursor).NextItem(0xc420159700, 0x0, 0x0, 0x0)
	/home/travis/gopath/src/github.com/v3io/v3io-tsdb/pkg/utils/asynciter.go:193 +0x5e0
github.com/v3io/v3io-tsdb/pkg/utils.(*AsyncItemsCursor).NextItem(0xc420159700, 0x0, 0x0, 0x0)
	/home/travis/gopath/src/github.com/v3io/v3io-tsdb/pkg/utils/asynciter.go:193 +0x5e0
github.com/v3io/v3io-tsdb/pkg/utils.(*AsyncItemsCursor).NextItem(0xc420159700, 0x0, 0x0, 0x0)
	/home/travis/gopath/src/github.com/v3io/v3io-tsdb/pkg/utils/asynciter.go:193 +0x5e0
github.com/v3io/v3io-tsdb/pkg/utils.(*AsyncItemsCursor).Next(0xc420159700, 0xc42029c748)
	/home/travis/gopath/src/github.com/v3io/v3io-tsdb/pkg/utils/asynciter.go:122 +0x2b
github.com/v3io/v3io-tsdb/pkg/querier.(*V3ioSeriesSet).Next(0xc420218820, 0xc42029c748)
	/home/travis/gopath/src/github.com/v3io/v3io-tsdb/pkg/querier/seriesset.go:88 +0x4a0
github.com/v3io/v3io-tsdb/pkg/tsdb/tsdbtest.ValidateRawData(0x97a0a0, 0xc42017e480, 0xc42007e800, 0xc42029c748, 0x8, 0x1664617f64a, 0x1665064ae4a, 0x9337a8)
	/home/travis/gopath/src/github.com/v3io/v3io-tsdb/pkg/tsdb/tsdbtest/tsdbtest.go:192 +0x4dc
github.com/v3io/v3io-tsdb/test/benchmark.BenchmarkIngest(0xc42017e480)
```

**Comment by gshatz** [comment]
reduce the cycle of the performance test - we shouldn't check performance there (use it for data correctness validation)

**Comment by imakhlin** [comment]
It hangs even with 30 cycles. The problem is with huge latency between the CI builder and the cloud environment. Lets deal with it in separate PR

File: test/benchmark/README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Define the following environment variables.
 > **Note:** You can also define variables locally in a script or in the command line from which the relevant command is executed.
 
 ```bash 
-    V3IO_CONF="$HOME/go/bin/v3io-custom.yaml"
+    V3IO_TSDB_CONFIG="$HOME/go/bin/v3io-custom.yaml"
```
Comment (Sharon-iguazio) [inline test/benchmark/README.md]: I think **v3io-custom.yaml** should be renamed to **v3io-tsdb-custom.yaml** or some other name that includes "tsdb" and more uniquely identifies the purpose of this file.

**Comment by Sharon-iguazio** [comment]
The **README.md** file still has some **"aggregator"** instances, including, for example, `aggregatorGranularity`, which has been renamed to `aggregationGranularity`.

**Comment by Sharon-iguazio** [comment]
You might want to rename the remaining `aggregator` instances in the code while you're at it (up to you):
```
$ git grep -ic aggregator
README.md:5
pkg/aggregate/aggregate.go:6
pkg/aggregate/aggregate_test.go:14
pkg/aggregate/functions.go:1
pkg/aggregate/iterator.go:3
pkg/tsdb/tsdbtest/testutils/schema.go:2
pkg/tsdb/v3iotsdb_integration_test.go:20
promtsdb/promtsdb.go:3
```
Some of the instances are in message texts, which would be risk-free to change.

File: test/benchmark/README.md (hunk 1 above)
Comment (Sharon-iguazio) [reply test/benchmark/README.md]: You've now renamed the file to **v3io-tsdb-custom.yaml** in commit 97c69b4 (good), but there are still references to the old name:
```
$ git grep -wlic "v3io-custom\.yaml"
test/benchmark/README.md
```

**Comment by Sharon-iguazio** [comment]
I believe this PR also handles [TSDB-34](http://jira.iguazeng.com:8080/browse/TSDB-34), so it would have been good to mention it in the PR summary (which should add a link to the PR from the JIRA ticket).

**Pull request merged**